WORKSPACE

```diff
@@ -21,3 +21,8 @@ http_archive(
 load("@google_bazel_common//:workspace_defs.bzl", "google_common_workspace_rules")
 
 google_common_workspace_rules()
+
+maven_jar(
```

Comment:
We're going to have to move this into github.com/google/bazel-common so that this works properly for out pom-generator

Reply:
Ping on this

Reply:
Is this for me?  Unsure what to do for bazel-common.

Reply:
Is this for me?  Unsure what to do for bazel-common.

Reply:
If you rebase, you should be able to remove this and installed of this target, and then replace the dep with `@google_bazel_common//third_party/java/slf4j_api`

Reply:
Sorry, my mistake, it's not ready just yet